.github/CONTRIBUTING.md

```diff
@@ -23,3 +23,22 @@ Say you have some documentation or code you want to add:
 0. Submit a Pull Request (PR)
 0. Wait for feedback or a merge!
 0. If your PR is open for a while, it may get out of synch with the master branch. In that case, pull the latest master branch and run `git rebase master` – while in your _feature_ branch – to update your feature branch to the latest master. You may need to use `git push --force-with-lease` to update your PR after rebasing.
+
+## PFE Pull Request Roles
+
+Please see [Github's docs](https://help.github.com/articles/reviewing-changes-in-pull-requests) for more info and details on reviewing changes in pull requests.
+
+Note: The use and implementation of Reviewers and Assignees for Panoptes Front End may be different than the way Reviewers and Assignees are used in other Zooniverse repos.
+
+### Reviewer
+- A Reviewer is a team member who provides thoughts, feedback, insight, or comments on a PR, but is not responsible for ensuring that the PR is ready for merging into the master branch.
+- If you add a general inline comment, you automatically are added as a reviewer. General PR comments do not do this. See Github's docs for more detail on this.
+- The developer creating the PR can optionally assign a Reviewer to the PR, if they desire their input on it, but it's ok for a PR to only have an Assignee and no Reviewer.
+- It's also ok for a PR to have the same person as both the Assignee and the Reviewer.
+- Also ok to have more than one Reviewer on a PR.
+
+### Assignee
+- The Assignee is the team member who deeply and thoroughly reviews the PR, ensuring that it is ready to merge into the Panoptes-Front-End master branch.
```

Comment:
Just noting that i was conflating my terms above (apologies) and my use of assignee has changed since github added the ability to request reviews.

I now exclusively use add reviewer to ask someone for their review and leave the assignee blank. I figure the assignee could add themselves to mark they are taking the PR on to signal to other team members that it's taken.